JapaneseFilter/sections/specific.txt

```diff
@@ -45,7 +45,9 @@ admin.blog.fc2.com###ad_sidebody
 admin.blog.fc2.com##.column-left_ad
 koro-pokemon.com###sadr
 ||appbank.net/wp-content/uploads/*/appape.png
-koro-pokemon.com##div[id^="ad"]
+biz-journal.jp,koro-pokemon.com##div[id^="ad"]
+biz-journal.jp###relationEntry > div[style="min-height: 100px;"]
+biz-journal.jp##.entryBody > div[style^="text-align:center;"][style$="font-weight:bold;"]
```

Comment:
This rule doesn't hit on my end. If it's for the placeholder middle in the body, it's `##.entryBody > div[style$="min-height:280px;"]` here.